phpcs.xml.dist

```diff
@@ -1,14 +1,22 @@
 <?xml version="1.0"?>
 <ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="PHP_CodeSniffer" xsi:noNamespaceSchemaLocation="phpcs.xsd">
+
+    <!-- Documentation: https://github.com/squizlabs/PHP_CodeSniffer -->
+
     <arg name="basepath" value="."/>
     <arg name="cache" value=".phpcs-cache"/>
     <arg name="colors"/>
     <arg name="extensions" value="php"/>
 
-    <rule ref="PSR12" />
-    <rule ref="Generic.PHP.RequireStrictTypes.MissingDeclaration" />
-    <rule ref="Generic.Arrays.DisallowLongArraySyntax" />
-
     <file>NetscapeBookmarkParser.php</file>
     <file>tests</file>
+
+    <rule ref="PSR12">
+        <!-- Remove line length warnings which will break ci pipeline -->
+        <exclude name="Generic.Files.LineLength"/>
```

Comment:
I think it's a better idea to disable this rule only for concerned lines instead of the whole codebase.
Something like: `// phpcs:disable Generic.Files.LineLength` (not sure for the syntax)